src/controllers/helpers.js

```diff
@@ -0,0 +1,104 @@
+const {
+  Author, Book, Reader, Genre,
+} = require('../models');
+
+const get404Error = (model) => ({ error: `The ${model} could not be found.` });
+
+const getModel = (model) => {
+  const models = {
+    book: Book,
+    reader: Reader,
+    genre: Genre,
+    author: Author,
+  };
+  return models[model];
+};
+
+const removePassword = (sentData) => {
+  delete sentData.password;
+
+  return sentData;
+};
+
+const chooseModelToInclude = (model) => {
+  if (model === 'book') return { include: [{ model: Genre }, { model: Author }] };
+  if (model === 'genre') return { include: Book };
```

Comment:
have you ever seen switch cases? If you ever have a lot more options, switch statements are good to use too, but you can use either method.